doc/api/webstreams.md

```diff
@@ -1467,16 +1493,61 @@ added: v16.7.0
 * Returns: {Promise} Fulfills with a {Blob} containing the full contents
   of the stream.
 
+```mjs
+import { blob } from 'node:stream/consumers';
+
+const dataBlob = new Blob(['hello world from consumers!']);
+
+const readable = dataBlob.stream();
+const data = await blob(readable);
+console.log(`from readable: ${data.size}`);
+```
+
+```cjs
+const { blob } = require('node:stream/consumers');
+
+const dataBlob = new Blob(['hello world from consumers!']);
+
+const readable = dataBlob.stream();
+blob(readable).then((data) => {
+  console.log(`from readable: ${data.size}`);
+});
+```
+
 #### `streamConsumers.buffer(stream)`
 
-<!-- YAML
-added: v16.7.0
--->
+\<!-added: v16.7.0
+\-->
```

Comment:
was this expected?

Reply:
Definitely not, I think my auto correction got messed up a bit, will correct in the next commit